packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -333,6 +340,19 @@ suite('createComponent', () => {
     assert.equal(el.firstElementChild!.localName, 'div');
   });
 
+  test('can set multiple children to multiple slots', async () => {
+    const children = window.React.createElement('div');
+    const foos = window.React.createElement('div');
+    const bars = window.React.createElement('div');
+
+    await renderReactComponent({children, foos, bars});
+
+    assert.equal(el.childNodes.length, 3);
+    assert.equal(el.children?.[0].getAttribute('slot'), null);
+    assert.equal(el.children?.[1].getAttribute('slot'), 'slot-a');
```

Comment:
These should be `display: contents` right? Let's test for that.

Reply:
Yah let's! Sounds good